app/build.gradle

```diff
@@ -47,14 +47,15 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
         coreLibraryDesugaringEnabled true
     }
     kotlinOptions {
-        jvmTarget = '1.8'
+        jvmTarget = '17'
     }
     buildFeatures {
+        buildConfig = true
```

Comment:
Btw why have you enabled build config? App doesn't really use build config anywhere

Reply:
If the build config is disabled, the following errors were encountered:
`app/src/main/java/com/starry/greenstash/ui/screens/settings/composables/AboutScreen.kt:58:30 Unresolved reference: BuildConfig`
and 
`app/src/main/java/com/starry/greenstash/ui/screens/input/composables/InputScreen.kt:128:30 Unresolved reference: BuildConfig`.

Reply:
Oh makes sense,  i think build configs were generated automatically on older AGP versions without explicitly enabling it in the `build.gradle` file, but yeah adding this line shouldn't cause any harm i guess 👍